e2e-polybft/framework/test-cluster.go

```diff
@@ -183,7 +183,7 @@ func WithMintableNativeToken(mintableToken bool) ClusterOption {
 	}
 }
 
-func WithSecretsCallback(fn func([]types.Address, *TestClusterConfig)) ClusterOption {
+func WithSecretsCallback(fn func([]types.Address, *TestClusterConfig, *TestCluster)) ClusterOption {
```

Comment:
We can access the `TestClusterConfig` via `TestCluster`, so I'm suggesting simplifying callback signature
```suggestion
func WithSecretsCallback(fn func([]types.Address, *TestCluster)) ClusterOption {
```

Reply:
fixed https://github.com/0xPolygon/polygon-edge/commit/3d92de70afa94706f3002144958fe9b5783ba13c